litezip/main.py

```diff
@@ -0,0 +1,108 @@
+# -*- coding: utf-8 -*-
+from collections import namedtuple
+
+from lxml import etree
+
+from .exceptions import MissingFile
+
+
+__all__ = (
+    'parse_collection',
+    'parse_litezip',
+    'parse_module',
+    'Collection',
+    'Module',
+)
+
+
+MODULE_FILENAME = 'index.cnxml'
+COLLECTION_FILENAME = 'collection.xml'
+
+Collection = namedtuple('Collection', 'id, file, resources')
+Module = namedtuple('Module', 'id, file, resources')
+
+
+COLLECTION_NSMAP = {
+    "bib": "http://bibtexml.sf.net/",
+    "c": "http://cnx.rice.edu/cnxml",
+    "col": "http://cnx.rice.edu/collxml",
+    "data": "http://www.w3.org/TR/html5/dom.html#custom-data-attribute",
+    "datadev": "http://dev.w3.org/html5/spec/#custom",
```

Comment:
This made me think of it, but isn't the right place for the code. If we see anything w/ the datadev namespace, we should figure out how to update it to use the data namespace.